codice/alliance
# CAL-253: Removed bower.

#### What does this PR do?

Removes bower.

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?

@brendan-hofmann 
@AzGoalie 
@bdeining  
@jlcsmith 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@stustison
@pklinef

#### How should this be tested?

- Pull down https://github.com/djblue/webjars/tree/DDF-2767 and build.
- Pull down the PR changes and do a quick build.
- Bring up all the uis and open the network tab in the devtools.
- Make sure everything loads.

NOTE: https://github.com/djblue/webjars/tree/DDF-2767 needs to be merged and released before this PR can be merged.

#### Any background context you want to provide?

Bower is causing issues with reproducible and stable builds. To fix this issue, bower is being removed and dependencies are being resolved using npm or webjars.

#### What are the relevant tickets?

[CAL-253](https://codice.atlassian.net/browse/CAL-253)
[DDF-2767](https://codice.atlassian.net/browse/DDF-2767)

#### Screenshots (if appropriate)
#### Checklist:
